Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift

```diff
@@ -103,6 +103,7 @@ class FormTextItemViewTests: XCTestCase {
             return false
         }
         
+        sut.isEditing = true
         sut.textField.delegate?.textFieldDidEndEditing?(sut.textField)
```

Comment:
maybe add color check somewhere? Like "Write the test that reproduces the bug you are fixing" 😄

Reply:
it was easy so I just added some assertions, Thanks!